plugins/workspace-minimap/src/minimap.ts

```diff
@@ -82,4 +86,37 @@ export class Minimap {
         this.minimapWorkspace.zoomToFit();
       });
     }
+
+    /**
+     * Changes the primary workspace viewport when clicking on the minimap
+     * @param event The minimap browser event.
+     */
+    private onClick(event: PointerEvent): void {
+      // Calculates the scale between the minimap and primary workspace.
+      const primaryMetrics = this.primaryWorkspace.getMetrics();
+      const contentDiv = this.minimapWorkspace.svgBlockCanvas_.
+          getBoundingClientRect();
```

Comment:
Could you use `getMetrics` here as well? I'm pretty sure that's equivalent to what you're doing. If it's possible I think it's better to use consistent properties, so then we know we're actually comparing the different sizes of the same thing!